R/calcGAINS2025forREMIND.R

```diff
@@ -0,0 +1,177 @@
+#' Calculates air pollutant emissions, activities and emission factors
+#' for all scenarios and SSPs available from GAINS, including both
+#' legacy and 2025 variants of GAINS data. This generates the actual
+#' files needed for REMIND in the proper format, for the scenario logic,
+#' see `GAINS2025scenarios`.
+#'
+#' @return Activity levels, emissions or emission factors
+#' @author Gabriel Abrahao
+#' @param subtype "emission_factors", "emissions","emissions_starting_values"
+#'
+#' @importFrom abind abind
+#' @importFrom utils head tail
+#' @importFrom magclass as.magpie
+#' @importFrom tidyr pivot_longer drop_na
```

Comment:
Some of these are not used in the functions after all and seem to be leftovers from development